addons/message_update_v2.16/manifest.json

```diff
@@ -50,12 +45,7 @@
       {
         "id": "extra_1_216",
```

Comment:
@flodolo I [updated this ID for translations yesterday](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/7216/commits/65a12d28565bc8fca8f5bd660a03863813532812), do I need to update it again

Reply:
Yes, and I'd like to understand how we avoid this level of churn.

This is the third request for this file that will get to localizers :-(

Reply:
This was a last minute string change, so, ideally, better planning 😅

I'm not sure how feasible this is, but it would be great if we could come up with a way to not push strings to localizers until they have been verified by QA possibly

Reply:
> This was a last minute string change, so, ideally, better planning 😅
> 
> I'm not sure how feasible this is, but it would be great if we could come up with a way to not push strings to localizers until they have been verified by QA possibly

You only need to let me know, that's a manual process (automation creates a pull request, I need to review it and merge it).

Note that we're already one week past string freeze, and we're still adding strings, so I assumed we needed that string out as soon as possible.

Reply:
It was a good assumption, and if you had asked me yesterday, I would've definitely given you the go-ahead anyways. Because it's only a one day difference, in terms the of release timelines, I think it's probably fine to go ahead with these new strings. If this was going to push the release out by weeks, than yea, we could definitely push back on these string changes.

I'm guessing the bigger downside here is that we don't want our volunteer translators wasting there time translating things we aren't going to use. If they haven't already been translated by localizers, is there a way to remove the unused strings, or mark them somehow as "do not translate"?

Reply:
The strings will be removed once we merge this, assuming they're not used in any other branch.

There's no way around it.